Firmware/RTK_Surveyor/SD.ino

```diff
@@ -63,9 +63,9 @@ bool sdPresent(void)
       if (response == 1) break;
     }
     if (response != 1) return (false); //Card failed to respond to idle
-  
-    return (true);
   }
+  
```

Comment:
I haven't seen hardware card detect yet. I thought it was going here...

Reply:
It will! Patience. I'll have to get the green wire out to test it!